src/FMI3/c.jl

```diff
@@ -355,7 +355,7 @@ function fmi3ExitInitializationMode(c::FMU3Instance)
         elseif fmi3IsScheduledExecution(c.fmu)
             c.state = fmi3InstanceStateClockActivationMode
         else
-            c.state = fmi3InstanceStateEventMode
+            c.state = fmi3InstanceStateContinuousTimeMode
```

Comment:
compliant to state machine in FMI3-sepc?

Reply:
nope, artifact of a test https://github.com/ThummeTo/FMIImport.jl/pull/70/commits/d44b18dde7be53f4325643f290ced1b8651ca530